src/Playground.res

```diff
@@ -891,23 +874,11 @@ module Settings = {
 
     let warnFlagTokens = WarningFlagDescription.Parser.parse(warn_flags)->Result.getOr([])
 
-    let onResetClick = evt => {
-      ReactEvent.Mouse.preventDefault(evt)
-
-      let open_modules = switch readyState.selected.apiVersion {
```

Comment:
This is gone? We still need this (for v11 at least).

Reply:
I renamed this callback to `onWarningFlagsResetClick` to make it clearer that this is triggered by clicking on `[reset]` in the 'Warnings flags' section of the settings panel.

Reply:
I removed the `open_modules` update part of this function as:
i) there's no way for users to configure open modules (at least as far as I can tell)
ii) the updating of `open_modules` on compiler load and switch is actually managed by the `useEffect` in `CompilerManagerHook`